tuxemon/db.py

```diff
@@ -57,7 +57,7 @@
 logger = logging.getLogger(__name__)
 
 # Load the default translator for data validation
-T.collect_languages(prepare.CONFIG.recompile_translations)
```

Comment:
as a style guideline, we should not be using `0` in place of False, and instead should use `False` when our intention is for the boolean value.

Reply:
Alright